ecyp-database.Rmd

```diff
@@ -198,19 +216,23 @@ ecyp_update <- reactive({
     print("Refresh was run")
 
     refresh_table(sheet_url) %>%
-      dplyr::filter(Unique_Number == enter_number())
+      dplyr::filter(`Unique Number` == enter_number()) %>% 
+      t() %>% 
+      data.frame()
   }
   if (!is.null(enter_number())) {
     refresh_table(sheet_url) %>%
-      dplyr::filter(Unique_Number == enter_number())
+      dplyr::filter(`Unique Number` == enter_number()) %>% 
+      t() %>% 
+      data.frame()
   }
 })
 
 DT::DTOutput("table")
 
 # Load the reactive values into data table
 output$table <- DT::renderDataTable({
-  ecyp_update()
+  datatable(ecyp_update(), options = list(pageLength = 1000, dom = "t"))
```

Comment:
This shuts off the pages of rows and instead shows all the users information on one page (I put 1000 because its such a ridiculously large number that all rows should always be included).

Reply:
Oh and the `dom = "t"` gets rid of some other table extras we didn't need. Makes it look cleaner.